ur_robot_driver/CMakeLists.txt

```diff
@@ -103,6 +103,7 @@ add_library(ur_robot_driver
     src/ur/dashboard_client.cpp
     src/ur/tool_communication.cpp
     src/rtde/rtde_writer.cpp
+    src/ros/data_field_publisher.cpp
```

Comment:
Please add this to the actual node and not to the non-ros library. Please also see #34 that might get merged before this.